.github/workflows/build-docs.yml

```diff
@@ -141,4 +141,4 @@ jobs:
             [**Click here**](${{ env.URL }}) to see the build log.
         env:
           URL: https://github.com/${{ github.repository_owner }}/${{ github.event.repository.name }}/actions/runs/${{ github.run_id }}?check_suite_focus=true
-        if: failure() && github.event_name == 'pull_request_target'
+        if: failure() && github.event_name == 'pull_request_target'
```

Comment:
Is there any change here? Do you want to unstage this file since the before and after diffs are identical?